project-api/ballerina-projects/src/main/java/io/ballerina/projects/PackageContext.java

```diff
@@ -99,4 +99,8 @@ ModuleContext defaultModuleContext() {
     BallerinaToml ballerinaToml() {
         return this.ballerinaToml;
     }
+
+    PackageConfig packageConfig() {
```

Comment:
We should not be exposing the packageConfig. Instead we expose individual fields (whichever is necessary). Anyway looks like there is only used in [Package#packagePath()](https://github.com/ballerina-platform/ballerina-lang/pull/25653/files#r488148846) and so we would not need this.

Reply:
Changed implementation to:
```java
Path packagePath() {
        return this.packageConfig.packagePath();
}
```